exercises/E-conditionals/exercise.js

```diff
@@ -8,6 +8,13 @@
 
 var name = "Daniel";
 var danielsRole = "mentor";
+if ( danielsRole )
```

Comment:
This should check specifically for the mentor role. You're effectively just checking if the string is a non null value which will always evaluate to true.

It should be:

if(danielsRole == 'mentor')